src/app/main/component/layout/components/header/header.component.spec.ts

```diff
@@ -48,7 +48,6 @@ describe('HeaderComponent', () => {
 
   let userServiceMock: UserService;
   userServiceMock = jasmine.createSpyObj('UserService', ['onLogout']);
-  userServiceMock.onLogout = () => true;
```

Comment:
why do we need to delete this ?

Reply:
Because we don't have such method anymore